pkg/infoutil/infoutil_linux.go

```diff
@@ -134,6 +134,6 @@ func mobySysInfo(info *dockercompat.Info) *sysinfo.SysInfo {
 		g := fmt.Sprintf("/user.slice/user-%d.slice", rootlessutil.ParentEUID())
 		mobySysInfoOpts = append(mobySysInfoOpts, sysinfo.WithCgroup2GroupPath(g))
 	}
-	mobySysInfo := sysinfo.New(true, mobySysInfoOpts...)
+	mobySysInfo := sysinfo.New(mobySysInfoOpts...)
```

Comment:
Why remove this?

Reply:
because of the docker API changes :crying_cat_face: 

> But introducing buildkit API would bring some dependency issue, for example, buildkit pinned docker with github.com/docker/docker => github.com/docker/docker v20.10.3-0.20220831131523-b5a0d7a188ac+incompatible, so we also have to pin on that version. I have tried to use v20.10.7 but it failed to compile because of some API changes.

I still have no other idea other than the current changes.